__docs__/wonder-blocks-tooltip/tooltip.argtypes.ts

```diff
@@ -1,14 +1,28 @@
 export default {
     placement: {
+        description:
+            "Where the tooltip should appear in relation to the anchor element. Defaults to `top`.",
         control: {
             type: "select",
             options: ["top", "bottom", "right", "left"],
         },
     },
-
     title: {
+        description: "Optional title for the tooltip content.",
         control: {
             type: "text",
         },
+        table: {
+            type: {
+                summary: "string",
+            },
+        },
+    },
+    contentStyle: {
+        description:
+            "Optional subset of CSS styles for the tooltip content. Currently supports `color` and `padding`.",
+    },
+    backgroundColor: {
+        description: "Optional background color for the tooltip content.",
```

Comment:
Can you add the `control` field to these? Currently, it doesn't let you put in your own values in the control panel in the `Default` example, but I think that's something we should be able to do. And having `table { type: }}` would also be good for showing the types, especially for `contentStyles` so people can see exactly which styles can be changed.